news/12551.trivial.rst

```diff
@@ -0,0 +1 @@
+Integrate ``sphinx-issues`` into the Sphinx config.
```

Comment:
By the way, if it's decided that this should go to a change log, it should probably link the integrated project with a hyperlink.